modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannels.testcase

```diff
@@ -527,6 +527,94 @@ definition {
 			locator1 = "CommerceEntry#ENTRY_NAME_BODY_VERTICAL_ELLIPSIS");
 	}
 
+	@description = "This is a test for COMMERCE-9459. CanAddEditAndRemoveTrackingURLToShippingMethod"
```

Comment:
Can you rename it to `CanEditShippingMethodTrackingURL`?

Reply:
Well, I could, yet I believe that just "Edit" doesn't reflect the actual test.